controllers/comments.controllers.js

```diff
@@ -17,4 +14,18 @@ function getCommentsByArticleId(req, res, next) {
         .catch(next);
 }
 
-module.exports = { getCommentsByArticleId };
+//task 7
+function postCommentByArticleId(req,res,next){
+    const {article_id} = req.params;
+    const {username,body} = req.body;
+    selectArticleById(article_id)
```

Comment:
you don't need to check that the article exists here as article_id is a foreign key so if you try insert the comment referencing an article that doesn't exist sql will throw an error as foreign keys need to be valid and exist